src/index.ts

```diff
@@ -182,7 +182,7 @@ export default class WebPhone {
         };
 
         options.sipErrorCodes = sipErrorCodes;
-        options.switchBackInterval = this.sipInfo.switchBackInterval;
+        options.switchBackInterval = this.sipInfo.switchBackInterval || 30;
```

Comment:
HI @vyshakhbabji . This parameter is in seconds, are you sure we want to switch back by default within 30 seconds? 
Second question: platform requirements if no switchBackInterval specified is to switch back after next sip provision update, so I think we do not need to set defaults here.

Reply:
This is 30 min ,  no ? 

    `let switchBackInterval = this.switchBackInterval ? this.switchBackInterval * 1000 : null;`

Until 19.2 release, there is no switchback interval provided. So, when the client goes through the reconnection attempt and connect to the backup proxy, there is no way for it to attempt to switch back and those connections stay back in the backup location ( had this issue while DR on GIG )